src/util/path.js

```diff
@@ -107,6 +107,20 @@
     }
   }
 
+  /**
+   * Calculates the coordinates of a vector from angle and distance
+   * @param {Number} angle the angle of the vector in radians
+   * @param {Number} distance the distance/length of the vector
+   * @return {Object.x} x coordinate of point
+   * @return {Object.y} y coordinate of point
+   */
+  function calcVectorPoint(angle, distance) {
+    var coords = {};
+    coords.x = fabric.util.cos(angle) * distance;
+    coords.y = fabric.util.sin(angle) * distance;
+    return coords;
+  }
```

Comment:
i do not feel good about this function.
The name is not really descriptive, and for now is really used just in your specific use case, i can't see a clear reusable case for it.
i would prefer doing this directly where are you using vec.x and vec.y.

```
raphemeInfo.renderLeft -= fabric.util.cos(angle) * offsetDist
```

Reply:
sure, I can change that

Reply:
done